AspNetCore/DotvvmAuthSample/DotvvmAuthSample/Startup.cs

```diff
@@ -47,10 +47,18 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientId = Configuration["AzureAD:ClientId"];
                 options.Authority = string.Format(Configuration["AzureAd:AadInstance"], Configuration["AzureAd:Tenant"]);
                 options.SignedOutRedirectUri = Configuration["AzureAd:PostLogoutRedirectUri"];
+                //options.ResponseType = "code";
```

Comment:
I'd change this to something like that:

```
// Uncomment the following section if you need "code" response type
// options.ResponseType = "code";
// options.ClientSecret = "Configuration["AzureAd:ClientSecret"];
```

Reply:
Good point, added this to the source code